src/MovieDetails/MovieDetails.js

```diff
@@ -2,29 +2,79 @@ import React, { Component } from 'react'
 import './MovieDetails.css'
 
 class MovieDetails extends Component {
-    constructor() {
-        super();
-        this.state = {}
+    constructor(props) {
+        super(props);
+        this.state = {
+            id: props.id,
+            backdrop_path: '',
+            average_rating: 0,
+            budget: 0,
+            genres: [],
+            overview: '',
+            poster_path: '',
+            release_date: '',
+            revenue: 0,
+            runtime: 0,
+            tagline: '',
+            title: '',
+            error: '',
+        }
     }
+
+    componentDidMount() {
+        fetch(`https://rancid-tomatillos.herokuapp.com/api/v2/movies/${this.state.id}`)
+        .then(response => response.json())
+        .then(data => this.setState(data.movie))
+        .catch(error => this.setState({error: error.message}))
+    }
+
+    getGenres() {
+        return this.state.genres.join(', ')
+    }
+
+    formatAmounts(amount) {
+        return amount.toLocaleString()
+    }
+
+    checkIfLoading() {
+        if (!this.state.title && !this.state.error) {
+            return <h1 className='error'>Loading...</h1>
+        }
+    }
+
+    handleIfFailed() {
+        if (this.state.error) {
+            return <h1 className='error'>Failed to load</h1>
+        }
+    }
+
     render() {
         return (
-            <section>
-                <div className='main-info'>
-                    <h1>Movie Title</h1>
-                    <div className='movie-details'>
-                        <p className='details'>Release Date</p>
-                        <p>Movie Genres</p>
-                    </div>
-                    <div className='movie-details'>
-                        <p className='details'>Budget</p>
-                        <p className='details'>Revenue</p>
-                        <p>Run Time</p>
-                    </div>
-                </div>
-                <p className='summary'>
-                    sdkasdnkask ldankk sdlnkl nask lna snkdkandkna kdaka sasdkas 
-                </p>
-            </section>
+            <div className='movies-div'>
+                {this.checkIfLoading()}
+                {this.handleIfFailed()}
+                {(!this.state.error && this.state.title) &&
+                    <section style={{background:
```

Comment:
Ooooh I'm not a fan of this inline styling. Can you tie this styling to a class instead??